samples/snippets/deletes/deletes_test.py

```diff
@@ -0,0 +1,144 @@
+    rows[1].set_cell("stats_summary", "connected_wifi", 1, timestamp)
+    rows[1].set_cell("stats_summary", "os_build", "PQ2A.190405.004", timestamp)
+    rows[1].set_cell("cell_plan", "data_plan_05gb", "true", timestamp)
+    rows[2].set_cell("stats_summary", "connected_cell", 0, timestamp)
+    rows[2].set_cell("stats_summary", "connected_wifi", 1, timestamp)
+    rows[2].set_cell("stats_summary", "os_build", "PQ2A.190406.000", timestamp)
+    rows[2].set_cell("cell_plan", "data_plan_05gb", "true", timestamp)
+    rows[3].set_cell("stats_summary", "connected_cell", 1, timestamp)
+    rows[3].set_cell("stats_summary", "connected_wifi", 1, timestamp)
+    rows[3].set_cell("stats_summary", "os_build", "PQ2A.190401.002", timestamp)
+    rows[3].set_cell("cell_plan", "data_plan_10gb", "true", timestamp)
+    rows[4].set_cell("stats_summary", "connected_cell", 1, timestamp)
+    rows[4].set_cell("stats_summary", "connected_wifi", 0, timestamp)
+    rows[4].set_cell("stats_summary", "os_build", "PQ2A.190406.000", timestamp)
+    rows[4].set_cell("cell_plan", "data_plan_10gb", "true", timestamp)
+
+    table.mutate_rows(rows)
+
+    # Ensure mutations have propagated.
+    row_set = RowSet()
+
+    for row_key in row_keys:
+        row_set.add_row_key(row_key)
+
+    fetched = list(table.read_rows(row_set=row_set))
+
+    while len(fetched) < len(rows):
+        time.sleep(5)
+        fetched = list(table.read_rows(row_set=row_set))
+
+    yield table_id
+
+
+def test_delete_from_column(capsys, snapshot, table_id):
+    deletes_snippets.delete_from_column_sample(PROJECT, BIGTABLE_INSTANCE, table_id)
+    out, _ = capsys.readouterr()
+    snapshot.assert_match(out)
+
+
+def test_delete_from_column_family(capsys, snapshot, table_id):
+    deletes_snippets.delete_from_column_family_sample(
+        PROJECT, BIGTABLE_INSTANCE, table_id
+    )
+    out, _ = capsys.readouterr()
+    snapshot.assert_match(out)
+
+
+def test_delete_from_row(capsys, snapshot, table_id):
+    deletes_snippets.delete_from_row_sample(PROJECT, BIGTABLE_INSTANCE, table_id)
+    out, _ = capsys.readouterr()
+    snapshot.assert_match(out)
+
+
+def test_streaming_and_batching(capsys, snapshot, table_id):
+    deletes_snippets.streaming_and_batching_sample(PROJECT, BIGTABLE_INSTANCE, table_id)
+    out, _ = capsys.readouterr()
+    snapshot.assert_match(out)
+
+
+def test_check_and_mutate(capsys, snapshot, table_id):
+    deletes_snippets.check_and_mutate_sample(PROJECT, BIGTABLE_INSTANCE, table_id)
+    out, _ = capsys.readouterr()
+    snapshot.assert_match(out)
+
+
+def test_drop_row_range(capsys, snapshot, table_id):
+    deletes_snippets.drop_row_range_sample(PROJECT, BIGTABLE_INSTANCE, table_id)
+    out, _ = capsys.readouterr()
+    snapshot.assert_match(out)
+
+
+def test_delete_column_family(capsys, snapshot, table_id):
+    deletes_snippets.delete_column_family_sample(PROJECT, BIGTABLE_INSTANCE, table_id)
+    out, _ = capsys.readouterr()
```

Comment:
I notice the snapshots at the end of the PR are empty. Ideally after each test I think we should print the data that remains in the table like we do in readSnippets. I think this just means adding the following code to the end of each function in `delete_snippets.js`.

```
for row in rows:
    print_row(row)
```

Reply:
Yeah I had the snapshots to print things out, but I think it was not necessary and noisy. I decided to go with calling the functions and checking that there is no errors.